src/util/storybook-docs/ColorSwatch.tsx

```diff
@@ -22,7 +23,7 @@ export interface CopyButtonProps {
 
 const CopyButton = ({ value, copyToClipboard }: CopyButtonProps): JSX.Element => (
   <button
-    onClick={() => (copyToClipboard ? copyToClipboard((value = '')) : null)}
+    onClick={() => (copyToClipboard ? copyToClipboard((value = '')) : noop)}
```

Comment:
the correct way is to set `noop` to be the default value of `copyToClipboard` in the props destructuring.